### PR TITLE
Default fuzz-checker standard to c++17.

### PR DESCRIPTION
**Pull request by Virv12**
Read TM_CXX_STD_VERSION.
Mark EXIT as [[noreturn]].

### CODE REVIEW

File: src/tools/fuzz_checker/fuzzer.cpp [hunk 1]
```diff
@@ -12,8 +12,12 @@
...
 int MAIN(int argc, char **argv);
 
-void EXIT(int status) {
-    throw status;
+struct Exit {
+    int status;
+};
+
+[[noreturn]] void EXIT(int status) {
+    throw Exit{status};
```
Comment (edomora97) [inline src/tools/fuzz_checker/fuzzer.cpp:20]: My only gripe about this PR is the spacing of these braces. I'm not that used to C++, have you used clang-format (with who knows which config)?

Comment (Virv12) [reply src/tools/fuzz_checker/fuzzer.cpp:20]: Out of 7 default configs for clang-format, four of them use `Exit{status}`, 2 use `Exit{ status }` and 1 uses `Exit { status }`.
Which do you prefer? xD

Comment (edomora97) [reply src/tools/fuzz_checker/fuzzer.cpp:20]: Lol! The one which is more in agreement with the rest of the file. If that too much effort no biggie, I'll merge this as it is.

Comment (Virv12) [reply src/tools/fuzz_checker/fuzzer.cpp:20]: The current version is consistent because this syntax is not used anywhere else.

**Review by edomora97 — APPROVED** [review]

**Pull request merged**